en/mkdocs.yml

```diff
@@ -2051,636 +1086,19 @@ plugins:
-              'tutorials/integration-tutorials/sap-integration.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/sap-integration'
-              'tutorials/integration-tutorials/sending-a-simple-message-to-a-datasource.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/sending-a-simple-message-to-a-datasource'
-              'tutorials/integration-tutorials/sending-a-simple-message-to-a-service.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/sending-a-simple-message-to-a-service'
-              'tutorials/integration-tutorials/service-catalog-tutorial-for-proxy-services.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/service-catalog-tutorial-for-proxy-services'
-              'tutorials/integration-tutorials/service-catalog-tutorial.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/service-catalog-tutorial'
-              'tutorials/integration-tutorials/storing-and-forwarding-messages.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/storing-and-forwarding-messages'
-              'tutorials/integration-tutorials/transforming-message-content.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/transforming-message-content'
-              'tutorials/integration-tutorials/using-inbound-endpoints.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/using-inbound-endpoints'
-              'tutorials/integration-tutorials/using-scheduled-tasks.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/using-scheduled-tasks'
-              'tutorials/integration-tutorials/using-templates.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/using-templates'
-              'tutorials/integration-tutorials/using-the-gmail-connector.md' : 'https://mi.docs.wso2.com/en/latest/learn/integration-tutorials/using-the-gmail-connector'
-
+              'design/prototype-api/create-a-mock-api-with-an-inline-script.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/create-a-prototype-api/'
+              'design/prototype-api/deploy-and-test-mock-apis.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/create-a-prototype-api/'
+              'design/prototype-api/create-a-prototype-api.md': 'https://apim.docs.wso2.com/en/4.2.0/design/prototype-api/overview/'
+              'deploy-and-publish/deploy-on-gateway/api-gateway/message-tracing.md': 'https://apim.docs.wso2.com/en/4.2.0/get-started/about-this-release/#removed-features-and-functionalities'
+              'develop/customizations/customizing-the-developer-portal/customize-api-listing/categorizing-and-grouping-apis/api-category-based-grouping.md': 'https://apim.docs.wso2.com/en/4.2.0/reference/customize-product/customizations/customizing-the-developer-portal/customize-api-listing/api-category-based-grouping/'
```

Comment:
The version within the path should be 4.4.0. Shall we revert the change done to these lines?